js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -49,7 +48,21 @@ type CountPerCodeLocation = {
   assetCount: number;
 };
 
-function buildAssetCountBySection(assets: AssetTableFragment[]): AssetCountsResult {
+type AssetDefinitionMetadata = {
+  definition: {
+    owners: Array<
+      {__typename: 'UserAssetOwner'; email: string} | {__typename: 'TeamAssetOwner'; team: string}
+    >;
+    computeKind: string | null;
+    groupName: string | null;
+    repository: {
+      name: string;
+      location: {name: string};
+    };
+  } | null;
+};
+
```

Comment:
I would go with something like this in order to keep a single source of truth for the base types.

```
type AssetDefinitionMetadata = {
  definition: Pick<AssetNode['definition'],  'owners' | ' computeKind' | 'groupName' | 'repository'>;
```
or if you don't like picking keys like that you could do 
```
type AssetDefinitionMetadata = {
  definition: {
    owners: AssetNode['definition']['owners']
    ...
  },
```

Reply:
good call, I added this